src/components/controller/controller.ts

```diff
@@ -0,0 +1,41 @@
+import AppLoader from './appLoader';
+
+class AppController extends AppLoader {
+    public getSources(callback: Function): void {
```

Comment:
It's not clear what type of callback the getSources method expects. Could you please expand typization of the getSources argument?